jaxrs/src/main/java/org/killbill/billing/jaxrs/json/SubscriptionJson.java

```diff
@@ -87,6 +88,7 @@ public static class EventSubscriptionJson extends JsonBase {
         private final UUID eventId;
         private final BillingPeriod billingPeriod;
         private final DateTime effectiveDate;
+        private final Date catalogEffectiveDate;
```

Comment:
The `equals` and `hashCode` methods would need to be updated for this additional field